src/language-service/src/schemas/integrations/conditions.ts

```diff
@@ -181,7 +181,7 @@ export interface StateCondition {
    * The state the entity (or entities) must have for this condition to pass.
    * https://www.home-assistant.io/docs/scripts/conditions/#state-condition
    */
-  state: State | State[];
```

Comment:
This is incorrect. States are strings, always.

Reply:
Done